build.gradle

```diff
@@ -0,0 +1,29 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath "com.palantir:gradle-flexversion:0.4.0"
+    }
+}
+
+apply plugin: "groovy"
+apply plugin: "idea"
+
+repositories {
+    jcenter()
+}
+
+apply plugin: "gradle-flexversion"
+flexversion.useTags = true
+group "com.palantir"
+version flexVersion()
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+
+    testCompile 'org.spockframework:spock-core:0.7-groovy-2.0@jar'
```

Comment:
double or single quotes? please pick one. also, we typically try to alphabetize these so that concurrent edits have less chance of conflicts